ops/terraform/modules/resources/cw_metric_alarms/main.tf

```diff
@@ -0,0 +1,92 @@
+locals {
+  alarm_actions       = var.alarm_notification_arn == null ? [] : [var.alarm_notification_arn]
+  ok_actions          = var.ok_notification_arn == null ? [] : [var.ok_notification_arn]
+}
+
+# HTTP 500 Errors
+resource "aws_cloudwatch_metric_alarm" "http-500" {
+  count = var.create_cw_alarms ? 1 : 0
+
+  alarm_name                = "${var.app}-${var.env}-http-500"
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
+  evaluation_periods        = "1"
+  period                    = "300"
+  statistic                 = "Maximum"
+  threshold                 = "0.0"
```

Comment:
Given a threshold of `0.0`, shouldn't the comparison operator be `GreaterThanThreshold`?

Reply:
Yes, thank you!